src/plugins/collapsibleColumns/collapsibleColumns.js

```diff
@@ -104,6 +104,20 @@ class CollapsibleColumns extends BasePlugin {
      * @type {EventManager}
      */
     this.eventManager = null;
+    /**
+     * List of currently collapsed columns.
+     *
+     * @private
+     * @type {Number[]}
+     */
+    this.collapsedColumns = [];
+    /**
+     * List of collapsable coords.
+     *
+     * @private
+     * @type {Object}
+     */
+    this.collapsableCoordsList = {};
```

Comment:
Here we shouldn't use literal object to store boolean values about collapsible coordinates. In the code below key uses to store that information are numeric type, and since the object cannot have a numeric key, it is converted to a string which can decrease access performance. The `Map` or `Set` data structure may be better to use in this case.

Reply:
If we have a configured plugin in `settings`, `this. collapsableCoordsList` have a reference to object `this.buttonEnabledList`.  So to be consistent we should change `buttonEnabledList` to `Map`. But sometimes we need iterate over this object and use our `objectEach` helper. After the change to `Map` we couldn't use this helper. 
And to be more consistent, object `this.collapsedSections` should be rewritten to `Map`, because this object also has the numeric key.
Should I change each these object to `Map`?

Reply:
I generally refer to the changes introduced in this PR. However, if you want to rewrite `this.collapsedSections` collection to `Map`, do not hesitate to do it. 

And if you want to iterate over `Map` you can use `arrayEach` helper, it supports the iterable protocol. Or you can use simple `for` loop to do it. It's up to you.